.github/workflows/newtag.yml

```diff
@@ -80,7 +81,7 @@ jobs:
         run: |
           cd ${{ env.repoName }}
           mvn -Ptemplate-validation-tests clean install --file weblogic-azure-vm/arm-oraclelinux-wls/pom.xml
-          mvn -Ptemplate-validation-tests clean install --file weblogic-azure-vm/arm-oraclelinux-wls-admin/pom.xml
+          mvn -Ptemplate-validation-tests clean install --file weblogic-azure-vm/arm-oraclelinux-wls-admin/pom.xml -Dgit.repo.owner=${{ env.repoOwner }} -Dgit.tag=${{ env.ref }}
```

Comment:
I need to see the same sanitization regarding the `ref` input as on the `buildWls4Vm*` workflows.  Also, please update the comment with the `cURL` command example so that shows the use of the inputs.

Reply:
Added sanitization and updated the cURL example. Code changes for other offers should in separated PRs